src/Controllers/Admin/NewsController.php

```diff
@@ -52,6 +54,7 @@ protected function showEdit(?News $news, bool $sendNotification = true, bool $is
                 'is_pinned'      => $news ? $news->is_pinned : false,
                 'is_highlighted' => $news ? $news->is_highlighted : false,
                 'send_notification' => $sendNotification,
+                'mails' => $mails,
```

Comment:
Eventually `notifications_count` would be clearer?

Reply:
done